routes/index.js

```diff
@@ -347,7 +347,7 @@ module.exports = () => {
 
 			Account.findOne({ username })
 				.then(account => {
-					if (account.wins + account.losses < 50) {
+					if (account.wins + account.losses + Math.min(20, account.winsPractice + account.lossesPractice || 0) < 50) {

```

Comment:
If either `account.winsPractice` or `account.lossesPractice` is undefined, `account.winsPractice + account.lossesPractice` is NaN, so the whole expression is 0.

Reply:
thats what || 0 is for but i forgot parens i think

Reply:
it's not parens, actually, you need `|| 0` after both `account.winsPractice` and `account.lossesPractice || 0`